device-types/Cisco/WS-C6506-E.yaml

```diff
@@ -8,14 +8,24 @@ is_full_depth: false
 console-ports:
   - name: con0
     type: rj-45
-power-ports:
-  - name: PS1
-    type: iec-60320-c20
-    maximum_draw: 8700
-  - name: PS2
-    type: iec-60320-c20
-    maximum_draw: 8700
 interfaces:
   - name: FastEthernet1
     type: 100base-tx
     mgmt_only: true
+module-bays:
```

Comment:
This looks like a parent device, is that so? If so, these also appear that it might have devices that slot into this chassis. If that is indeed true, then you should have all applicable device slots under `device-bays` rather than `module-bays` which are reserved for things like PSUs, Fans, SFP interface cards, etc. I don't use these so I do not know, but it looks like this should have device bays

Reply:
6500 is a modular switch